corehq/apps/reports/templatetags/xform_tags.py

```diff
@@ -35,7 +35,7 @@
 
 @register.simple_tag
 def render_form_xml(form):
-    xml = form.get_xml()
+    xml = form.get_xml().encode('utf-8')
```

Comment:
Do you mean the python type `unicode` or the actual encoding?
since utf-8 and ascii are indistinguishable for ascii characters I assumed this was safe for strings with no unicode characters and didn't break any forms on my local setup.